src/Plaid/PlaidClient.cs

```diff
@@ -278,13 +303,14 @@ public sealed class PlaidClient
 		/// <param name="request">The request.</param>
 		/// <returns>Task&lt;Management.StripeTokenResponse&gt;.</returns>
 		public Task<Management.StripeTokenResponse> FetchStripeTokenAsync(Management.StripeTokenRequest request) =>
-			PostAsync<Management.StripeTokenResponse>("processor/stripe/bank_account_token/create", request);
+			ParseResponse<Management.StripeTokenResponse>(
+				PostAsync("processor/stripe/bank_account_token/create", request));
 
 		#endregion
 
 		#region Private Members
 
-		private async Task<TResponse> PostAsync<TResponse>(string path, RequestBase request) where TResponse : ResponseBase, new()
+		private Task<HttpResponseMessage> PostAsync<TRequest>(string path, TRequest request) where TRequest : RequestBase
```

Comment:
This is named 'PostAsync' but it's not marked async and it awaits Async calls within so it's synchronous right? ... therefore naming convention says should just be 'Post' vs 'PostAsync'?

Reply:
Technically, the Async() trailing is more about whether the consumer should await or not, i.e. does it return a Task? Even though this method is itself technically synchronous, it calls an Async method and returns _it's_ Task, so it's still in the realm of async. I do need to update ParseResponse to ParseResponseAsync() though.